### ISSUE TITLE
Is it possible to include 'title' option to 3D chart?

### ISSUE THREAD

**Issue by rsaungc** [almende/vis#536]
Like network node chart, can we include 'title' (info will display upon mouse-over) for 3D chart?  Currently, 3D chart required 'int' for 'x', 'y' and 'z' and not other data type.  Tooltip will only print location number.   It will be helpful to have a 'title' option if data definition needs to display besides number.  Following table as an example, I would like to print title 

<p><tt><b>[x][y] = [NAME]</b></tt>
<p><tt><b>[1][1] = BOB</b></tt>
<p><tt><b>[1][2] = CARL</b></tt>
<p><tt><b>[3][3] = JOE</b></tt>

etc. when mouse over to certain block in 3D chart.

<table>
<tr>
<td><tt>Axis -> </td>
<td><tt>x:</td>
<td><tt>y:</td>
<td><tt>z:</td>
</tr>
<tr>
<td>NAME</td>
<td>DAY</td>
<td>PROJECT</td>
<td>COST</td>
</tr>

<tr>
<td>BOB</td>
<td>1</td>
<td>1</td>
<td>100</td>
</tr>

<tr>
<td>CARL</td>
<td>1</td>
<td>2</td>
<td>150</td>
</tr>

<tr>
<td>JOE</td>
<td>1</td>
<td>3</td>
<td>110</td>
</tr>

<tr>
<td>BOB</td>
<td>2</td>
<td>1</td>
<td>140</td>
</tr>

<tr>
<td>CARL</td>
<td>2</td>
<td>2</td>
<td>120</td>
</tr>

<tr>
<td>JOE</td>
<td>2</td>
<td>3</td>
<td>120</td>
</tr>

<tr>
<td>BOB</td>
<td>3</td>
<td>1</td>
<td>130</td>
</tr>

<tr>
<td>CARL</td>
<td>3</td>
<td>2</td>
<td>110</td>
</tr>

<tr>
<td>JOE</td>
<td>3</td>
<td>3</td>
<td>140</td>
</tr>

</table>

Thanks!

**Comment by AlexDM0** [comment]
Hi,

I don't believe this is currently possible, but it would be a nice addition to the graph3d. I'll put it in as a feature request.

Cheers!

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
